user.py

```diff
@@ -128,7 +137,7 @@ def user_profile(user_id, current_year):
 
     return render_template('user_profile.html', requested_user=requested_user, supervisors=all_users,
                            researcher=researcher, courses=courses, preferences=preferences, current_user=current_user,
-                           current_year=current_year)
+                           current_year=current_year, selected_supervisor=selected_supervisor)
```

Comment:
`selected_supervisor` is not present in user_profile.html nor search_supervisor.html you include in it. Where is it used and why ?

Reply:
Indeed there are fetched in the template : https://github.com/UCL-INGI/ictm-teaching/pull/40/files#diff-6ce7dbe16f41f3395d3ae4bd6acd7059769aece6a94dca3eb45446662c7227b2R25